repl/scala-2.11/src/main/scala/org/apache/spark/repl/SparkILoop.scala

```diff
@@ -19,7 +19,9 @@ package org.apache.spark.repl
 
 import java.io.BufferedReader
 
+// scalastyle:off println
 import scala.Predef.{println => _, _}
+// scalastyle:on println
```

Comment:
Nit: This looks a bit weird.

Reply:
This actually looks valid though. If I manually add ` import scala.Predef.{println => _, _}` somewhere not here, for example, `SQLConf` in the current master:

```
[error] .../spark/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala:26:21: Are you sure you want to println? If yes, wrap the code block with
[error]       // scalastyle:off println
[error]       println(...)
[error]       // scalastyle:on println
[error] .../spark/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala:26:0: scala.Predef. is in wrong order relative to scala.collection.immutable.
```

It looks recognising this as an error. Looks 1.0.0 fixes an issue about those style checking and detection. We might have to fix `println` token checker rule but I guess this should be orthogonal anyway.

Reply:
I said it's weird because this obviously not a place to print out something. Not much harm actually.